fronts-client/src/components/inputs/HoverActionButtonWrapper.tsx

```diff
@@ -45,6 +45,7 @@ interface WrapperProps {
 	toolTipAlign: 'left' | 'center' | 'right';
 	urlPath: string | undefined;
 	renderButtons: (renderProps: ButtonProps) => JSX.Element;
+	noPinboard?: boolean;
```

Comment:
Calling this `noPinboard` feels like a bit of a double negative: `noPinboard={true}` takes me a second to think about. Could we make it `showPinboard` or `enablePinboard`?

Reply:
You could have the prop default to `true` if not specified, no?